model/jpa/src/main/java/org/keycloak/events/jpa/JpaEventStoreProvider.java

```diff
@@ -69,8 +75,38 @@ public void clear(String realmId) {
     }
 
     @Override
-    public void clear(String realmId, long olderThan) {
```

Comment:
Could this method be left in the interface?

Reply:
I wonder what would be the point to keep the method in the interface and not actually use it the code. Or do you suggest to change the PR in a way that the method will be used?

Reply:
Thanks @vramik, good point. In the new storage, we will likely defer back to the cleanup per realm. Since the used event store implementation may differ from realm to realm, the event store should offer method to cleanup per realm.

Reply:
Thank you for explanation